test/test_component/test_overload.cpp

```diff
@@ -0,0 +1,19 @@
+#include "pch.h"
+#include "winrt/test_component.h"
+
+// Simple compile-only test to validate overloads coming from overridable interfaces compile.
+
+using namespace winrt;
+using namespace test_component;
+
+struct DerivedClass : OverloadClassT<DerivedClass>
+{
+	void Foo()
```

Comment:
[nitpick] This file is using tabs whereas all of the other files are using spaces.

(It would be nice if we had a .editorconfig and/or clang-tidy formatting so this is automatic)

Reply:
Tried adding an .editorconfig but there are too many preexisting inconsistencies in this repo that fixing them would bloat this PR. Would be good for another cleanup PR.

E.g. this is with just
```editorconfig
[*.{cpp,h}]
indent_style = space
indent_size = 4
```
![image](https://github.com/user-attachments/assets/3ba3a015-a225-4cef-b3e7-3edfc1bed299)

I'll fix this specific file since it's new.

Reply:
I agree that any sort of cleanup would belong in another PR.  I filed an issue to track this: #1459.